argocon-demos/2023-leveraging-argo-workflowtemplates-within-a-data-platform/README.md

```diff
@@ -0,0 +1,24 @@
+# Leveraging Argo WorkflowTemplates Within a Data Platform
+
+[![Pipekit Logo](https://raw.githubusercontent.com/pipekit/talk-demos/main/assets/images/pipekit-logo.png)](https://pipekit.io)
+
+[Leveraging Argo WorkflowTemplates Within a Data Platform - J.P. Zivalich, Pipekit & Yao Lin, Bloomberg](https://sched.co/1Jo9d)
+
+## The talk
+The talk [can be found here](https://youtu.be/g8TCgnrM648?feature=shared).
```

Comment:
nit. you can drop the `?feature=shared` and it'll still work

Reply:
Dropped